akka-typed/src/main/scala/akka/typed/receptionist/Receptionist.scala

```diff
@@ -0,0 +1,103 @@
+package akka.typed.receptionist
+
+import akka.typed.ActorRef
+import akka.typed.ActorSystem
+import akka.typed.Extension
+import akka.typed.ExtensionId
+import akka.typed.internal.receptionist.LocalReceptionist
+
+import scala.concurrent.duration._
+import scala.reflect.ClassTag
+
+class ReceptionistExt(system: ActorSystem[_]) extends Extension {
+  val receptionist: ActorRef[Receptionist.Command] =
```

Comment:
would it read nicer if it was `Receptionist().ref ! ...`?

Reply:
👍

Reply:
I had `ref` when I saw that the previous version had `receptionist`... But will change it back ;)